Library/Homebrew/cmd/audit.rb

```diff
@@ -1187,7 +1198,7 @@ def audit_urls
         problem "Don't use #{$1}use_mirror in SourceForge urls (url is #{p})."
       end
 
-      if p =~ /\/download$/
+      if p =~ %r{/download$}
```

Comment:
Nope, perfectly happy to go with `end_with`. Just wanted to get Rubocop off my back 😉